src/main/java/edu/harvard/hul/ois/fits/tools/ToolBelt.java

```diff
@@ -168,12 +168,12 @@ private Tool createToolClassInstance(Class<?> toolClass, Fits fits) throws Refle
 		try {
 			Constructor<?> ctor = toolClass.getConstructor(Fits.class);
 			instanceOfTheClass = ctor.newInstance(fits);
-			logger.debug("1-arg constructor for instantiating tool class: " + toolClass.getName());
+			logger.debug("1-arg constructor for instantiating tool class: {}", toolClass.getName());
 		} catch (NoSuchMethodException e) {
 			// now try a no-arg constructor
-			logger.warn("No Fits 1-arg constructor for tool class: " + toolClass.getName() + " -- trying no-arg constructor. Error message: " + e.getMessage());
+			logger.debug("No Fits 1-arg constructor for tool class: {}", toolClass.getName() + " -- trying no-arg constructor. Error message: " + e.getMessage());

```

Comment:
This line is not quite right. It should be:

```java
logger.debug("No Fits 1-arg constructor for tool class: {} -- trying no-arg constructor. Error message: {}", toolClass.getName(), e.getMessage());
```

With the template as the first arg and all of the values to substitute a the end

Reply:
Done. Should have given it a closer look. Thanks.